# Fix build for OpenSUSE users using system direct memory access.
# Imported from OpenSUSE patches at https://build.opensuse.org/package/show/network:chromium/chromium

--- a/ui/gfx/linux/client_native_pixmap_dmabuf.cc
+++ b/ui/gfx/linux/client_native_pixmap_dmabuf.cc
@@ -17,9 +17,6 @@
 #include "base/strings/stringprintf.h"
 #include "base/trace_event/trace_event.h"
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
-#include <linux/dma-buf.h>
-#else
 #include <linux/types.h>
 
 struct dma_buf_sync {
@@ -34,7 +31,6 @@ struct dma_buf_sync {
 
 #define DMA_BUF_BASE 'b'
 #define DMA_BUF_IOCTL_SYNC _IOW(DMA_BUF_BASE, 0, struct dma_buf_sync)
-#endif
 
 namespace gfx {
 
